### PR TITLE
Buildtools changes to support building the android runtime

### PR DESCRIPTION
**Pull request by mrvoorhe**

### CODE REVIEW

**Comment by joshpeterson** [comment]
LGTM

File: build.pl [hunk 1]
```diff
@@ -157,14 +190,21 @@
...
 	push @configureparams, "--disable-nls";  #this removes the dependency on gettext package
-	push @configureparams, "--prefix=$monoprefix";
-	push @configureparams, "--with-monotouch=no";
+	push @configureparams, "--disable-btls";  #this removes the dependency on cmake to build btls for now
 	push @configureparams, "--with-mcs-docs=no";
+	push @configureparams, "--prefix=$monoprefix";
+
+	if ($isDesktopBuild)
+	{
+		push @configureparams, "--with-monotouch=no";
```
Comment (joncham) [inline build.pl:206]: So we do build monotouch profile on android? Or `disableMcs` prevents that?

File: build.pl [hunk 2]
```diff
@@ -415,32 +734,55 @@
...
-		
+
 		print("\n>>> Calling make\n");
 		system("make $mcs -j$jobs") eq 0 or die ('Failed to make\n');
 		
-		print("\n>>> Calling make install\n");
-		system("make install") eq 0 or die ("Failed to make install\n");
+		if ($isDesktopBuild)
+		{
+			print("\n>>> Calling make install\n");
```
Comment (joncham) [inline build.pl:750]: Why is this tied to desktop build? I might prefer an explicit variable to control this

**Comment by joncham** [comment]
LGTM

File: build.pl (hunk 1 above)
Comment (mrvoorhe) [reply build.pl:206]: disableMcs prevents any class libs from being built.

File: build.pl (hunk 2 above)
Comment (mrvoorhe) [reply build.pl:750]: Keep reading, there's a big comment in the else block explaining the situation.

**Pull request merged**